Subject: [PATCH] darwin: enable CURLOPT_PINNEDPUBLICKEY

Enable use of the CURLOPT_PINNEDPUBLICKEY option on darwin
when running the tests.

Signed-off-by: Kyle J. McKay <mackyle@gmail.com>

---
 tests/runtests.pl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/runtests.pl b/tests/runtests.pl
index 6def307d..b87e8766 100755
--- a/tests/runtests.pl
+++ b/tests/runtests.pl
@@ -2366,6 +2366,7 @@ sub checksystem {
            }
            elsif ($libcurl =~ /securetransport/i) {
                $has_darwinssl=1;
+               $has_sslpinning=1;
                $ssllib="DarwinSSL";
            }
            elsif ($libcurl =~ /BoringSSL/i) {
---
